### PR TITLE
Move this-week-in-open-source into its own page

### PR DESCRIPTION
**Pull request by marcoow**
This moves the this-week-in-open-source posts into their own section on the website so they don't end up spamming the blog but we keep them for showing out contributions (and SEO).

[Preview](https://deploy-preview-1968--objective-northcutt-37494c.netlify.app/this-week-in-open-source/)

| List | Individual Update |
|-----|------------------|
| localhost_8080_this-week-in-open-source_ | localhost_8080_this-week-in-open-source_2022_11_24_ |

The markdown files are now kept in a separate directory and do not require any metadata anymore so the script for generating them will require a (presumably simple) update. /cc @BobrImperator

### CODE REVIEW

File: .percy.js [hunk 1]
```diff
@@ -1,14 +1,15 @@
 module.exports = {
   version: 2,
   static: {
-    // don't snapshot all of the blog
+    // don't snapshot neither all of the blog not all of this-week-in-open-source
```
Comment (zeppelin) [inline .percy.js]:
```suggestion
    // don't snapshot neither all of the blog nor all of this-week-in-open-source
```

File: src/this-week-in-open-source.njk [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+description: "Each week, we publish an update of our team's open source contributions."
+layout: "base.njk"
+---
+
+{% from "color-hero.njk" import colorHero %}
+{% from "twios-card.njk" import twiosCard %}
+
+{% set 'content' = {
+"title": "This week in Open Source at Mainmatter",
+"text": "Our software engineers are all active members of the open-source community and enjoy collaborating on various projects. Each week, we compile a summary of the team's latest contributions.",
```
Comment (SaeLLo) [inline src/this-week-in-open-source.njk]: We should decide if we write "open-source" or "open source". Right now it is a mix.

Comment (marcoow) [reply src/this-week-in-open-source.njk]: good point – should be _"open source"_

File: src/components/layouts/twios-post.njk [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+{% set post = collections.twios | getCollectionItem(page) %}
+<article>
+  <div class="post__header">
+    <div class="container container--lg">
+      <div class="post__meta">
+        <div class="post__date text--xs">
+          {{ page.date | monthDayYear }}
+        </div>
+      </div>
+      <h1 class="post__title">This week in Open Source at Mainmatter #{{ post.data.number }}</h1>
```
Comment (SaeLLo) [inline src/components/layouts/twios-post.njk]: Due to stringency, "week" should be written with capital letters.

File: src/twios/2022-02-07.md [hunk 4]
```diff
@@ -1,17 +1,4 @@
 ---
```
Comment (BobrImperator) [inline src/twios/2022-02-07.md]: What about these?

```suggestion
```

File: src/twios/2022-02-07.md [hunk 5]
```diff
@@ -1,17 +1,4 @@
...
-bio: "The Mainmatter team"
-description:
-  "A collection of work that our engineers have been carrying out in open-source
-  in the past few weeks."
-og:
-  image: /assets/images/posts/2022-02-07-this-week-in-os-4/og-image.jpg
-tagline: |
-  <p>Our software engineers are all active members of the open-source community and
-  enjoy collaborating on various projects. In this blog post, we have collected
-  some of the work they have done the past week!</p>
```
Comment (BobrImperator) [inline src/twios/2022-02-07.md:14]: Regarding the mention
> The markdown files are now kept in a separate directory and do not require any metadata anymore so the script for generating them will require a (presumably simple) update. /cc @BobrImperator

The script wasn't generating this, so there might be no needed changes.

Comment (marcoow) [reply src/twios/2022-02-07.md:14]: ok, great 👍

**Review by zeppelin — COMMENTED** [review]
I don't know eleventy that much, but apart from the typo, QA seems OK 👍

File: src/twios/2022-02-07.md (hunk 4 above)
Comment (marcoow) [reply src/twios/2022-02-07.md]: I thought those were needed somehow but turns out they aren't – removed

**Review by davidhajba — APPROVED** [review]

**Pull request merged**